vnet/lib/vnet/endpoints/1.0/responses/translation_static_address.rb

```diff
@@ -10,6 +10,12 @@ def self.generate(object)
 
         route_link = object.batch.route_link.commit
         res[:route_link_uuid] = route_link.uuid if route_link
+
+        i_network = object.batch.ingress_network.commit
+        e_network = object.batch.egress_network.commit
+
+        res[:ingress_network_uuid] = i_network.uuid if i_network
+        res[:egress_network_uuid] = e_network.uuid if e_network
```

Comment:
This ends up calling vnmgr twice. Should instead check if object.foo_network_id is valid, and then do batch.foo_network.canonical_uuid.commit.